test/RB_SQL/RB_mri_upload.sql

```diff
@@ -1,6 +1,6 @@
 SET FOREIGN_KEY_CHECKS=0;
 TRUNCATE TABLE `mri_upload`;
 LOCK TABLES `mri_upload` WRITE;
-INSERT INTO `mri_upload` (`UploadID`, `UploadedBy`, `UploadDate`, `UploadLocation`, `DecompressedLocation`, `InsertionComplete`, `Inserting`, `PatientName`, `number_of_mincInserted`, `number_of_mincCreated`, `TarchiveID`, `SessionID`, `IsCandidateInfoValidated`, `IsTarchiveValidated`, `IsPhantom`) VALUES (126,'cecile','2024-08-30 14:26:07','/data/incoming/MTL001_300001_V2_t1w.tgz','/data/not_backed_up/ImagingUpload-14-30-FoTt1K',0,0,'MTL001_300001_V2',NULL,NULL,74,NULL,1,0,'N');
+INSERT INTO `mri_upload` (`UploadID`, `UploadedBy`, `UploadDate`, `UploadLocation`, `DecompressedLocation`, `InsertionComplete`, `Inserting`, `PatientName`, `number_of_mincInserted`, `number_of_mincCreated`, `TarchiveID`, `SessionID`, `IsCandidateInfoValidated`, `IsTarchiveValidated`, `IsPhantom`) VALUES (126,'cecile','2024-08-30 14:26:07','/data/incoming/MTL001_300001_V2_t1w.tgz','/data/not_backed_up/ImagingUpload-14-30-FoTt1K',0,0,'MTL001_300001_V2',NULL,NULL,74,NULL,0,0,'N');
```

Comment:
@maximemulder I changed the column `IsCandidateInfoValidated` to set it to `0` instead of `1` because subject validation is supposed to happen later on. But on the test, it keeps complaining that `IsCandidateInfoValidated` is set to True. Do you have any idea why? Is there some sort of caching happening?

Reply:
right, I had that revelation in the middle of the night. I guess I just needed to get away from my computer and stop obsessing about this 🤣 Thanks for the prompt reply though!